_pages/software-tools/zoom.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Zoom
+---
+
+GSA offers [Zoom for Government](https://zoomgov.com/).
+
+## Setup
+
+1. Install the Zoom application through the Self Service app.
+1. When signing in, select `.zoomgov.com`.
+1. Consider [adding a profile picture and your location and pronouns to your display name](https://www.zoomgov.com/profile).
+
+## How to host a meeting
+
+[General instructions.](https://support.zoom.us/hc/en-us/articles/201362613-How-Do-I-Host-A-Video-Meeting-)
+
+### For meetings longer than 40 minutes
+
+[Ask](#questions) to be upgraded to a [Licensed user](https://support.zoom.us/hc/en-us/articles/201363173-Account-Types#usertype).
+
+## Rules
+
+Per a `User Action Required for Zoom Usage` email from GSA IT on 4/13/20:
+
+> **only Zoom for Government may be used for Zoom meetings initiated by GSA employees.**
+>
+> …
+>
+> **GSA employees may join Zoom meetings hosted by external partners**.
+>
+> - GSA employees may participate in Zoom meetings hosted by external partners **UNLESS** they are required to accept Zoom Terms of Service (TOS) or install software.
+> - …
+> - Zoom meetings hosted by external partners are to be limited to **non-sensitive** discussions.
+>
+> **Zoom Usage Guidance for GSA Hosted Meetings**
+>
+> - All Zoom meetings hosted by GSA require a meeting password.
+> - Do not use a Personal Meeting ID (PMI) for Zoom meetings hosted by GSA.
+> - Do not share links to Zoom meetings or classrooms via publicly available social media outlets. Provide the links directly to specific invitees.
+> - For public meetings hosted by GSA, limit screen sharing to "Host Only".
+
+## Questions?
+
+Reach out to paul.h.powell@gsa.gov, tennille.spence@gsa.gov, and joseph.drain@gsa.gov for help.
```

Comment:
Minor thing, but since our general guidance is to avoid personal emails, is there an office/channel/non-personal email we can direct folks to? If not, It's not a blocker.

Reply:
Not that I'm aware of, but I'll ask / suggest they do so and edit as a follow-up.